SUMMARY.md

```diff
@@ -23,11 +23,11 @@
     * [Lifecycle API reference](using-mobile-extensions/mobile-core/lifecycle/lifecycle-api-reference.md)
     * [Lifecycle metrics](using-mobile-extensions/mobile-core/lifecycle/lifecycle-metrics.md)
     * [Configuration keys](using-mobile-extensions/mobile-core/lifecycle/configuration-keys.md)
-  * [Signal](using-mobile-extensions/mobile-core/signals/README.md)
-    * [Signal API reference](using-mobile-extensions/mobile-core/signals/signal-api-reference.md)
   * [Rules Engine](using-mobile-extensions/mobile-core/rules-engine/README.md)
     * [Rules Engine technical details](using-mobile-extensions/mobile-core/rules-engine/rules-engine-details.md)
     * [Rules Engine consequence details](using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details.md)
+  * [Signal](using-mobile-extensions/mobile-core/signals/README.md)
+    * [Signal API reference](using-mobile-extensions/mobile-core/signals/signal-api-reference.md)
```

Comment:
We'll want to keep these in their original position. They are in the order users would typically configure them.

Reply:
Ahh ok, I thought it was a wonky alphabetical order. I'll go fix that back then.